github/github_test.go

```diff
@@ -0,0 +1,84 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var SearchQueryNoDate = SearchQuery{
+	Repo:        "CapstoneLabs/slick",
+	Labels:      []string{"bug"},
+	ClosedSince: "",
+}
+
+var SearchQueryWithDate = SearchQuery{
+	Repo:        "CapstoneLabs/slick",
+	Labels:      []string{"bug"},
+	ClosedSince: "2018-10-03T18:16:21Z",
+}
+
+var user = GHUser{Login: "3sky"}
+
+var IssueEventClose = IssueEvent{
+	Actor: user,
+	Event: "closed",
+}
+var IssueItemNoNumber = IssueItem{
+	Assignee: user,
+	Events:   []IssueEvent{IssueEventClose},
+}
+
+var IssueEventOpen = IssueEvent{
+	Actor: user,
+	Event: "open",
+}
+
+var IssueItemWithNumber = IssueItem{
+	Number:   1,
+	Assignee: user,
+	Events:   []IssueEvent{IssueEventOpen},
+}
+
+var config = Conf{
+	Repos: []string{"3sky/slick"},
```

Comment:
This should most likely be CapstoneLabs/slick so the test doesn't fail if you delete your fork.

```suggestion
	Repos: []string{"CapstoneLabs/slick"},
```

Reply:
My mistake, I thought I corrected it before PR.